### ISSUE TITLE
LOSS可视化（LOSS visualization）

### ISSUE THREAD

**Issue by m250317460** [qqwweee/keras-yolo3#688]
这里想问一下大家我们应该如何把训练过程中的LOSS参数可视化呢？
Here I would like to ask everyone how should we visualize the LOSS parameters during training?

**Comment by koujunxia** [comment]
您好，您解决了么？我也遇到了同样的问题。
